simulator/func_sim/operation.h

```diff
@@ -39,7 +39,7 @@ enum class Imm : uint8
 {
     NO, SHIFT,
     LOGIC, ARITH, TRAP, ADDR,
-    JUMP
+    JUMP, C_ARITH, C_JUMP
```

Comment:
I don't feel that we need an additional types here — both types should be good with ARITH type (signed decimal).

Reply:
I think C_ARITH is essential, because immediate of this type has 6th bit as sign bit. And I use C_JUMP instead of ARITH, because we don't need an extra comma in disassembly. What do you think?